tests/libvmi/storage.go

```diff
@@ -71,6 +95,12 @@ func addVolume(vmi *v1.VirtualMachineInstance, volume v1.Volume) {
 	}
 }
 
+func addFilesystem(vmi *v1.VirtualMachineInstance, filesystem v1.Filesystem) {
+	if !filesystemExists(vmi, filesystem) {
```

Comment:
I dislike this as it gives a false feeling that I constructed my VM as I wanted and didn't make mistakes.

Reply:
@brybacki Also mentioned this above. I implemented it like the other `addXYZ` functions for now but it should be changed in another PR.

Reply:
I am lacking reasoning why it should be changed in another PR. To me, this is similar to saying: Let me introduce a bug because there is a similar bug already and hopefully someone will fix both of them in x days.

Reply:
I intended to be that someone who will fix it in a follow-up PR but might as well do it for `addFilesystem` in this PR. See also https://github.com/kubevirt/kubevirt/pull/8348 for the other ones.

Reply:
better to fix the others in a new PR as we might discover some problems